### ISSUE TITLE
What does yank() return?

### ISSUE THREAD

**Issue by juangomezduaso** [r-lib/rray#129]
Documentation says : " a 1D vector is always returned."
 But a 1D rray is what we get.
Is it the desired behavior? Wouldn't it be better to return a bare vector?

**Comment by DavisVaughan** [comment]
Since `rray_extract()` is heavily related and always returns a 1D object, it would also have to return a vector.

**Comment by juangomezduaso** [comment]
Yes, I think that in both functions the most useful behavior  is probably to "get rid" of the container (on a value which had lost any dimensional meanig anyway)

**Comment by DavisVaughan** [comment]
yup thats what i ended up with too!

**Comment by DavisVaughan** [comment]
`rray_extract` is waiting on an xtensor bug fix but then it will do that

**Comment by DavisVaughan** [comment]
I think this is worth revisiting (or at least asking more questions about) since vctrs has this behavior of keeping the container type:

``` r
vctrs::new_vctr(1:5)[[2]]
#> <vctrs_vctr[1]>
#> [1] 2
```

<sup>Created on 2019-06-24 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by juangomezduaso** [comment]
I would rather return a **vctrs_vctr** .
 I dont see it as a "container" in the same way a **rray**  or a tibble are, but as a the equivalent of the bare vector in the  tidyverse.
